api/v1alpha1/fio_types.go

```diff
@@ -56,6 +56,7 @@ type FioSpec struct {
 
 	// PodConfig contains the configuration for the benchmark pod, including
 	// pod labels and scheduling policies (affinity, toleration, node selector...)
+	// +optional
 	PodConfig PodConfigurationSpec `json:"podConfig,omitempty"`
```

Comment:
we need to convince @julio-santiesteban to inline this so that we are consistent across all the benchmarks :)